### PR TITLE
Add cover page of application management and fix menu tree

### PR DESCRIPTION
**Pull request by journey2w**
### Bugs Fixed ###

- Add cover page of application management and fix menu tree.
- fix tidl.md and rpc-port.md.
- Change unicode characters to ascii.
- Fix a code block error that broke the layout of i18n page.

Reference: PR #346

### CODE REVIEW

**Comment by journey2w** [comment]
@pjh9216 
I couldn't assign you as a reviewer, please confirm tidl.md and rpc-port.md.

**Comment by pjh9216** [comment]
I think tidl.md and rpc-port.md are fine

File: docs/application/native/guides/connectivity/vpn.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The main features of the VPN Service API include:
   The VPN profile provides different functions for routing management, DNS management, and firewall management:
 
   - The `vpnsvc_init()` function is used to a initialize the VPN interface and get the handle.
-  - The `vpnsvc_h` handle is used to [configure the tunnel interface](#config) before using the [VPN application control](../app-management/common-appcontrol.md#vpnservice) to ask the user permission to connect to the VPN service.
+  - The `vpnsvc_h` handle is used to [configure the tunnel interface](#config) before using the [VPN application control](../app-management/common-appcontrols.md#vpnservice) to ask the user permission to connect to the VPN service.
```
Comment (mijin85cho) [inline docs/application/native/guides/connectivity/vpn.md]: It looks like the \<a name\> tags are removed, and we are using links generated by \# only.
`vpnservice` is old tag and removed now. To jump to the right location, it should be `vpn-service`.

File: docs/application/native/guides/connectivity/vpn.md [hunk 2]
```diff
@@ -213,7 +213,7 @@ To set the interface parameters:
 
 To configure the interface and connect to the VPN service through an application control:
 
-1. Define a function to launch the [VPN application control](../app-management/common-appcontrol.md#vpnservice) and a callback to handle the results:
+1. Define a function to launch the [VPN application control](../app-management/common-appcontrols.md#vpnservice) and a callback to handle the results:
```
Comment (mijin85cho) [inline docs/application/native/guides/connectivity/vpn.md]: It looks like the \<a name\> tags are removed, and we are using links generated by \# only.
`vpnservice` is old tag and removed now. To jump to the right location, it should be `vpn-service`.

File: docs/application/native/guides/app-management/app-controls.md [hunk 3]
```diff
@@ -666,24 +666,24 @@ The Tizen common application controls specify a standard protocol for sharing ap
...
-	- [Settings for Wi-Fi](common-appcontrol.md#settings_wifi)
-- [Voice Recorder](common-appcontrol.md#voice)
-- [VPN Service](common-appcontrol.md#vpnservice)
+- [Browser](common-appcontrols.md#browser)
+- [Calendar](common-appcontrols.md#calendar)
+- [Call](common-appcontrols.md#call)
+- [Camera](common-appcontrols.md#camera)
+- [Contact](common-appcontrols.md#contact)
+- [Email](common-appcontrols.md#email)
+- [File Storage](common-appcontrols.md#file)
```
Comment (mijin85cho) [inline docs/application/native/guides/app-management/app-controls.md]: It looks like the \<a name\> tags are removed, and we are using links generated by \# only.
`file` is old tag and removed now. To jump to the right location, it should be `file-storage`.

File: docs/application/native/guides/app-management/app-controls.md [hunk 4]
```diff
@@ -666,24 +666,24 @@ The Tizen common application controls specify a standard protocol for sharing ap
...
+- [Input Delegator](common-appcontrols.md#inputdelegator)
```
Comment (mijin85cho) [inline docs/application/native/guides/app-management/app-controls.md]: It looks like the \<a name\> tags are removed, and we are using links generated by \# only.
`inputdelegator` is old tag and removed now. To jump to the right location, it should be `input-delegator`.

File: docs/application/native/guides/app-management/app-controls.md [hunk 5]
```diff
@@ -666,24 +666,24 @@ The Tizen common application controls specify a standard protocol for sharing ap
...
+- [Map](common-appcontrols.md#map)
+- [Message](common-appcontrols.md#message)
+- [Multimedia](common-appcontrols.md#multimedia)
+- [System Settings](common-appcontrols.md#settings_main)
```
Comment (mijin85cho) [inline docs/application/native/guides/app-management/app-controls.md]: It looks like the \<a name\> tags are removed, and we are using links generated by \# only.
`settings_main` is old tag and removed now. To jump to the right location, it should be `system-settings`.

File: docs/application/native/guides/app-management/app-controls.md [hunk 6]
```diff
@@ -666,24 +666,24 @@ The Tizen common application controls specify a standard protocol for sharing ap
...
+	- [Settings for Bluetooth](common-appcontrols.md#settings_bluetooth)
```
Comment (mijin85cho) [inline docs/application/native/guides/app-management/app-controls.md]: It looks like the \<a name\> tags are removed, and we are using links generated by \# only.
`settings_bluetooth` is old tag and removed now. To jump to the right location, it should be `settings-for-bluetooth`.

File: docs/application/native/guides/app-management/app-controls.md [hunk 7]
```diff
@@ -666,24 +666,24 @@ The Tizen common application controls specify a standard protocol for sharing ap
...
+	- [Settings for Location](common-appcontrols.md#settings_location)
```
Comment (mijin85cho) [inline docs/application/native/guides/app-management/app-controls.md]: It looks like the \<a name\> tags are removed, and we are using links generated by \# only.
`settings_location` is old tag and removed now. To jump to the right location, it should be `settings-for-location`.

File: docs/application/native/guides/app-management/app-controls.md [hunk 8]
```diff
@@ -666,24 +666,24 @@ The Tizen common application controls specify a standard protocol for sharing ap
...
+	- [Settings for NFC](common-appcontrols.md#settings_nfc)
```
Comment (mijin85cho) [inline docs/application/native/guides/app-management/app-controls.md]: It looks like the \<a name\> tags are removed, and we are using links generated by \# only.
`settings_nfc` is old tag and removed now. To jump to the right location, it should be `settings-for-nfc`.

File: docs/application/native/guides/app-management/app-controls.md [hunk 9]
```diff
@@ -666,24 +666,24 @@ The Tizen common application controls specify a standard protocol for sharing ap
...
+	- [Settings for Wi-Fi](common-appcontrols.md#settings_wifi)
```
Comment (mijin85cho) [inline docs/application/native/guides/app-management/app-controls.md]: It looks like the \<a name\> tags are removed, and we are using links generated by \# only.
`settings_wifi` is old tag and removed now. To jump to the right location, it should be `settings-for-wi-fi`.

File: docs/application/native/guides/app-management/app-controls.md [hunk 10]
```diff
@@ -666,24 +666,24 @@ The Tizen common application controls specify a standard protocol for sharing ap
...
+- [Voice Recorder](common-appcontrols.md#voice)
```
Comment (mijin85cho) [inline docs/application/native/guides/app-management/app-controls.md]: It looks like the \<a name\> tags are removed, and we are using links generated by \# only.
`voice` is old tag and removed now. To jump to the right location, it should be `voice-recorder`.

File: docs/application/native/guides/app-management/app-controls.md [hunk 11]
```diff
@@ -666,24 +666,24 @@ The Tizen common application controls specify a standard protocol for sharing ap
...
+- [VPN Service](common-appcontrols.md#vpnservice)
```
Comment (mijin85cho) [inline docs/application/native/guides/app-management/app-controls.md]: It looks like the \<a name\> tags are removed, and we are using links generated by \# only.
`vpnservice` is old tag and removed now. To jump to the right location, it should be `vpn-service`.

**Comment by mijin85cho** [comment]
Thanks for the PR.

**Comment by mijin80lee** [comment]
TD Sync Completed.

Check the below file only. 
docs/application/native/guides/app-management/app-info-control.md
docs/application/native/guides/app-management/app-communication.md

**Pull request merged**